frontend/src/Activity/History/Details/HistoryDetails.js

```diff
@@ -207,7 +207,7 @@ function HistoryDetails(props) {
         reasonMessage = 'File was deleted by via UI';
         break;
       case 'MissingFromDisk':
-        reasonMessage = 'Sonarr was unable to find the file on disk so it was removed';
+        reasonMessage = 'Sonarr was unable to find the file on disk so the file was removed from the database';
```

Comment:
I think: 
`Sonarr was unable to find the file on disk so the file was unlinked from the episode in the database`

is the most accurate we can be, since the file itself was never in the database

Reply:
agreed; pushed